src/style.css

```diff
@@ -2,2 +1,85 @@
+* {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+}
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work.